content/docs/04.5-devx/05-app-profile.md

```diff
@@ -41,8 +42,18 @@ To create your App Profile,
 |Description (optional)   | Description of the App Profile, if any | 
 |Tag (optional)               | Tags on a cluster group are propagated to the cloud/datacenter environments.|
 
-3. **Add Service**: Select a service to start the configuration of the App Profile from the available services and configure the service per the use case.
+3. **Add Service**: Select a service to start the configuration of the App Profile from the available services and configure the service per the use case. The users can go for the following key services:
 
+  * Container Deployment: Containers are methods of building, packaging, and deploying an application. A container includes the code, runtime, libraries, and all the dependencies required by a containerized workload. Container deployment deploys containers to their target environment.
+ 
+
+ * [Helm](/devx/registries#palettehelmregistry): Palette App profile Pack from Helm registry charts. The public Palette Pack registry and a few popular Helm chart repositories are already available out of the box. Add additional pack registries or public or private chart registries to Palette.
+ 
+
+ * Manifest: Layers can be constructed using raw manifests to provision Kubernetes resources unavailable via Palette or Charts. Pack Manifests provide a pass-through mechanism wherein additional Kubernetes resources can be orchestrated onto a cluster and the rest of the stack. Specific integrations may require the creation of Secrets or CustomResourceDefinition (CRDs); manifest files can be attached to the layer to achieve this.
+  
+
+ *  Database Services: Different database applications such as MongoDB, PostgreSQl, Redis, etc.
```

Comment:
Between steps 3 and 4 we agreed to add a Warning box about namespace. Below I offered a couple variations of text for you to consider because you're more familiar with the product. Also, should "namespace" be bold?

_You suggested:_ "When adding layers to the App profile using manifest files, specify a **namespace** name to which the resource must belong."

_Another suggestion with slight change:_ "When adding layers to the App profile using manifest files, you must specify a **namespace** name to which the resource belongs."

_Another suggestion:_ "When adding layers to the App profile using manifest files, you must specify a **namespace** name to indicate which resource the layer belongs to."